tests/topotests/lib/topotest.py

```diff
@@ -1642,8 +1642,6 @@ def startRouter(self, tgen=None):
         # TODO remove the following lines after all tests are migrated to Topogen.
         # Try to find relevant old logfiles in /tmp and delete them
         map(os.remove, glob.glob("{}/{}/*.log".format(self.logdir, self.name)))
-        # Remove old core files
```

Comment:
We still want to remove the core files on the first startRouter or start of the test I think.

Reply:
Also maybe we also don't want to do the other cleanup in here when a test is stop/starting the routers instead of staring up.

Reply:
the directory is deleted so it's not a problem